client/src/components/timeline/revision/revision.js

```diff
@@ -62,7 +62,7 @@ class Revision extends React.Component {
 
 		if ( this.props.duration ) {
 			displayTimelapse = (
-				<Timelapse icon="timelapse" date={this.props.duration.humanize()} />
+				<Timelapse icon="timelapse" samePage={this.props.samePage} date={this.props.duration.humanize()} />
```

Comment:
Not directly related to this change, I don't see "icon" being defined or used in the TimelineTimelapse component, what am I missing?

Reply:
good catch! that prop is no longer used, I'll remove it. :)